$MirOS: ports/print/cups/patches/patch-notifier_Makefile,v 1.1 2006/06/21 19:25:35 bsiegert Exp $
--- notifier/Makefile.orig	Mon Jul 28 23:14:20 2008
+++ notifier/Makefile	Sun Oct 26 22:57:13 2008
@@ -78,7 +78,7 @@ depend:
 
 mailto:	mailto.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o mailto mailto.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o mailto mailto.o $(LIBS)
 
 
 #
@@ -87,7 +87,7 @@ mailto:	mailto.o ../cups/$(LIBCUPS)
 
 rss:	rss.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o rss rss.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o rss rss.o $(LIBS)
 
 
 #
@@ -96,7 +96,7 @@ rss:	rss.o ../cups/$(LIBCUPS)
 
 testnotify:	testnotify.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o testnotify testnotify.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o testnotify testnotify.o $(LIBS)
 
 
 $(OBJS):	../Makedefs
